src/Softeq.NetKit.Chat.Domain.Services/DomainServices/MemberService.cs

```diff
@@ -12,9 +12,12 @@
 using Softeq.NetKit.Chat.Domain.Services.Mappings;
 using Softeq.NetKit.Chat.Domain.Services.Utility;
 using Softeq.NetKit.Chat.Domain.TransportModels.Request.Member;
```

Comment:
check if we really need to have both Softeq.NetKit.Chat.Domain.TransportModels and Softeq.NetKit.Chat.TransportModels  namespaces